src/test.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
We should probably remove test.txt and root-folder src instead of simply merging it into dev

Reply:
Wouldn't it be better to merge it first and then use a "fix" to solve it in the dev branch. Otherwise, we have to delete it first in main and there should actually no more direct commit happen.

Reply:
Fair enough, good point!